.travis.yml

```diff
@@ -0,0 +1,2 @@
+language: nix
+script: nix-channel --remove nixpkgs && nix-channel --add https://nixos.org/channels/nixos-16.09/ nixpkgs && nix-channel --update && nix-build release.nix
```

Comment:
We don't need this anymore.  This was for before we learned how to pin `nixpkgs` without the `NIX_PATH`.  See: https://nixos.wiki/wiki/How_to_fetch_Nixpkgs_with_an_empty_NIX_PATH

If you use the above recipe then the only command you need here is `nix-build release.nix`

Reply:
How do you integrate the script with the `nixpkgs.json` parsing here? https://github.com/awakesecurity/proto3-suite/blob/master/release.nix#L88

Maybe something like this?
``` 
 nixpkgs =
    let
      json = builtins.fromJSON (builtins.readFile ./nixpkgs.json);
    in fetchNixpkgs { rev = json.rev; sha = json.sha; };
```

Reply:
That won't work because the `sha256` that `fetchNixpkgs` expects is computed in a different way.  `gRPC-haskell` has a good example of instructions on how to do this: https://github.com/awakesecurity/gRPC-haskell/blob/master/nixpkgs.nix